components/html-block/html-block.js

```diff
@@ -242,6 +245,10 @@ class HtmlBlock extends LitElement {
 		}
 	}
 
+	async getLoadingComplete() {
```

Comment:
This is to ensure that MathJax has loaded and done its typesetting before snapshotting. Previously, I suspect this was "just working" because all the tests were in the same page and by the time it got down to the math ones, a few seconds had passed.

I found that even waiting until after all the `render()` calls resolve wasn't enough, and that a further `100ms` was needed for the first one. The way I've buried this is by creating an optional `getLoadingComplete` on the renderer plugins. The MathJax one returns a Promise that's just a `100ms` timeout the _first_ time.